Beta_Integrations/ProofpointServerProtection/ProofpointServerProtection.yml

```diff
@@ -124,7 +124,7 @@ script:
     - contextPath: Proofpoint.SmartSearch.MessageID
       type: string
       description: Search results email message ID
-    - contextPath: Proofpoint.SmartSearch.Proofpoint.SmartSearch.Suborg
+    - contextPath: Proofpoint.SmartSearch.Suborg
```

Comment:
trigger a build without the BC check using `Utils/trigger_content_build_with_no_backward_check.sh`